### PR TITLE
document on code review

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

File: doc/code-review.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Code is merged by zuul, but before a pull request is merged, zuul runs all project specific jobs of the so called 'gate pipeline'. If all jobs are successful, code is merged. If a job fails, the pull request will get a comment added and the pipeline finishs without merging.
+
+The requirements and triggers for the gate pipeline are defined by [zuul-config's gate pipeline](https://github.com/thoth-station/zuul-test-config/blob/master/zuul.d/_pipelines.yaml#L45-L55):
+
+### Requirements
+
+The following requirements must be met to start merging a Pull Request:
+
+* a 'approved' [GitHub review](https://help.github.com/articles/about-pull-request-reviews/) must exists
+* the 'approved' [label](https://help.github.com/articles/about-labels/) must be set
```
Comment (fridex) [inline doc/code-review.md]: I guess this one is not needed anymore.

File: playbooks/deprovision.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+- name: Playbook to provision Thoth Core
```
Comment (fridex) [inline playbooks/deprovision.yaml]: De-provision?

File: playbooks/deprovision.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  gather_facts: false
+  connection: local
+
+  tasks:
+  - fail: msg="Bailing out. this play requires 'THOTH_NAMESPACE'"
+    when: THOTH_NAMESPACE is not defined
+
+  - name: "deleting Thoth Templates"
+    command: "oc delete template {{ item }} --namespace {{ THOTH_NAMESPACE }}"
```
Comment (fridex) [inline playbooks/deprovision.yaml]: What about delete all the stuff just based on label?

File: doc/code-review.md (hunk 1 above)
Comment (goern) [reply doc/code-review.md]: so how do we want the process to be? just an approved label? just a approved review?

if we go with the approved review, we need all auto update PRs to be approved...

File: playbooks/provision.yaml [hunk 4]
```diff
@@ -7,11 +7,11 @@
...
-    THOTH_BACKEND_NAMESPACE: "{{ lookup('env','USER') }}-thoth-dev"
-    THOTH_MIDDLETIER_NAMESPACE: "{{ lookup('env','USER') }}-thoth-dev"
-    THOTH_FRONTEND_NAMESPACE: "{{ lookup('env','USER') }}-thoth-dev"
-    THOTH_INFRA_NAMESPACE: "{{ lookup('env','USER') }}-thoth-dev"
-    THOTH_DEPLOYMENT_NAME: "{{ lookup('env','USER') }}-thoth-dev"
+    THOTH_BACKEND_NAMESPACE: "thoth-test-core"
+    THOTH_MIDDLETIER_NAMESPACE: "thoth-test-core"
+    THOTH_FRONTEND_NAMESPACE: "thoth-test-core"
+    THOTH_INFRA_NAMESPACE: "thoth-test-core"
+    THOTH_DEPLOYMENT_NAME: "thoth-test-core"
```
Comment (fridex) [inline playbooks/provision.yaml]: I would like to keep env vars here not to accidentally do changes on existing deployment (and if the deployment exists, it is user's deployment).

Comment (goern) [reply playbooks/provision.yaml]: then we should use some general env var rather than an user specific dev environment?!

**Pull request closed without merging**